compiler-cli/src/publish.rs

```diff
@@ -131,6 +88,98 @@ The repository URL {} returned status {}
     }
 }
 
+fn check_for_name_squatting(package: &Package) -> Result<(), Error> {
+    if package.modules.len() > 1 {
```

Comment:
I don't remember, does this count test modules as well? Because if those are counted as well then every default package would come with two modules by default: the `<name>` and `<name>_test` one

Reply:
It doesn't in this case because it has been compiled in prod mode for publishing